### PR TITLE
Scpsl 12.0.2 pluginapi beta

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Door.cs [hunk 1]
```diff
@@ -427,7 +426,14 @@ public bool BreakDoor(DoorDamageType type = DoorDamageType.ServerCommand)
...
         /// </summary>
         /// <returns><see langword="true"/> if the door was able to be pried open.</returns>
-        public bool TryPryOpen() => Base is PryableDoor pryable && pryable.TryPryGate();
+        public bool TryPryOpen() => Base is PryableDoor pryable && pryable.TryPryGate(null);
+
+        /// <summary>
+        /// Tries to pry the door open. No effect if the door cannot be pried.
+        /// </summary>
+        /// <returns><see langword="true"/> if the door was able to be pried open.</returns>
+        /// <param name="player">The amount of damage to deal.</param>
```
Comment (Thundermaker300) [inline Exiled.API/Features/Door.cs:435]: Update docs for player parameter

File: Exiled.Events/Patches/Events/Scp914/UpgradingPlayer.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ namespace Exiled.Events.Patches.Events.Scp914
     ///     Patches <see cref="Scp914Upgrader.ProcessPlayer(ReferenceHub, bool, bool, Vector3, Scp914KnobSetting)" />
     ///     to add the <see cref="Scp914.UpgradingPlayer" /> event.
     /// </summary>
-    [HarmonyPatch(typeof(Scp914Upgrader), nameof(Scp914Upgrader.ProcessPlayer))]
+    // [HarmonyPatch(typeof(Scp914Upgrader), nameof(Scp914Upgrader.ProcessPlayer))]
```
Comment (louis1706) [inline Exiled.Events/Patches/Events/Scp914/UpgradingPlayer.cs:32]: Require fix

File: Exiled.Events/Patches/Events/Player/UsingRadioBattery.cs [hunk 3]
```diff
@@ -25,7 +25,7 @@ namespace Exiled.Events.Patches.Events.Player
     ///     Patches <see cref="RadioItem.Update" />.
     ///     Adds the <see cref="Handlers.Player.UsingRadioBattery" /> event.
     /// </summary>
-    [HarmonyPatch(typeof(RadioItem), nameof(RadioItem.Update))]
+    // [HarmonyPatch(typeof(RadioItem), nameof(RadioItem.Update))]
```
Comment (louis1706) [inline Exiled.Events/Patches/Events/Player/UsingRadioBattery.cs:28]: Require Fix

**Comment by louis1706** [comment]
We need to test all the Scp096 event before merging (Merge only when  SCPSLpluginAPI will being merge to major branch)

File: Exiled.API/Features/Door.cs (hunk 1 above)
Comment (louis1706) [reply Exiled.API/Features/Door.cs:435]: will be fix later

**Pull request merged**